platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/com.peregrine.replication.impl.LocalFileSystemReplicationService-local.xml

```diff
@@ -31,7 +31,7 @@
           targetFolder="${sling.home}/staticreplication"
           creationStrategy="1"
           exportExtensions="[data.json=per:Page|per:Template,infinity.json=per:Object,html=per:Page|per:Template,*~raw=nt:file,css=per:Site|per:FeLib,js=per:Site|per:FeLib]"
-          mandatoryRenditions="thumbnail.png"
+          mandatoryRenditions=""
```

Comment:
That's just a little extra that slipped in but actually has little to do with the PR.
We dont need peregrine to generate thumbnails when publishing assets since we're use cloudflare images to transform them.